src/Text/Pandoc/SelfContained.hs

```diff
@@ -258,12 +258,10 @@ combineSvgAttrs svgAttrs imgAttrs =
   dropPointZero t = case T.stripSuffix ".0" t of
                        Nothing -> t
                        Just t' -> t'
-  combinedAttrs = concat [
-    imgAttrs,
+  combinedAttrs = replaceMaybe mergedClasses $ imgAttrs ++
```

Comment:
Instead of using `replaceMaybe` I'd suggest getting rid of that function and just added `"class"` to the list at ln. 264 and also use `[kv | (k,v) <- imgAttrs | k /= "class"]`.

But isn't there another possible source of duplicated attributes?  For example, suppose `imgAttrs` contains `widdth` and so does `svgAttrs`; won't we get two `width` attributes?

Reply:
> Instead of using `replaceMaybe` I'd suggest getting rid of that function and just added `"class"` to the list at ln. 264 and also use `[kv | (k,v) <- imgAttrs | k /= "class"]`.

Thanks - my Haskell is (very visibly) rusty, and I forgot about comprehensions.

> But isn't there another possible source of duplicated attributes? For example, suppose imgAttrs contains widdth and so does svgAttrs; won't we get two width attributes?

Yes, I believe we will. I had a narrow fix in mind for the `class` attribute since the other ones don't seem to have a natural way to merge the data. I can make further changes, but I don't know what you would prefer:

- let the outer attribute win
- let the inner attribute win
- issue a warning
- issue an error